GUI/GUIListPanel.cpp

```diff
@@ -437,10 +437,14 @@ void GUIListPanel::OnMouseDown(int X, int Y, int Buttons, int Modifier) {
 void GUIListPanel::OnMouseWheelChange(int x, int y, int modifier, int mouseWheelChange) {
 	if (!m_MouseScroll) {
 		return;
-	} else if (m_VertScroll->_GetVisible() && m_VertScroll->PointInside(x, y)) {
+	}
+	else if ( (PointInsideList(x, y) && !m_MultiSelect) || (m_VertScroll->_GetVisible() && m_VertScroll->PointInside(x, y)) )
```

Comment:
Generally the projects aims to do K&R style brackets-on-the-same-line formatting, i.e:

```cpp
if (test) {
    foo();
} else {
    bar();
}
```

Anywhere doing brackets on new lines is legacy code.

Reply:
Yup, was orienting by such legacy code.
On it.